main.tf

```diff
@@ -105,6 +105,7 @@ resource "aws_db_instance" "db_including_name" {
   db_subnet_group_name        = "${local.db_subnet_group_name}"
   engine                      = "${var.engine_type}"
   engine_version              = "${var.engine_version}"
+  final_snapshot_identifier   = "${var.name}"
```

Comment:
I don't think this field is documented in the README, mind including it if that's alright?

Reply:
Oh ignore, misread